Gopkg.toml

```diff
@@ -46,5 +38,40 @@ required = ["sigs.k8s.io/testing_frameworks/integration",
   name = "k8s.io/client-go"
   version = "kubernetes-1.10.1"
 
+[[constraint]]
+  name = "github.com/onsi/ginkgo"
+  version = "v1.5.0"
+
+[[constraint]]
+  name = "github.com/onsi/gomega"
+  version = "v1.4.0"
+
+[[constraint]]
+  name = "github.com/ghodss/yaml"
+  version = "1.0.0"
+
+[[constraint]]
+  name = "go.uber.org/zap"
+  version = "1.8.0"
+
+[[constraint]]
+  name = "sigs.k8s.io/testing_frameworks"
+  revision = "f53464b8b84b4507805a0b033a8377b225163fea"
+
+[[constraint]]
+  name = "github.com/go-logr/logr"
+  revision = "9fb12b3b21c5415d16ac18dc5cd42c1cfdd40c4e"
+
+[[constraint]]
```

Comment:
@DirectXMan12 Why you think we should stay away from `glog`?

Reply:
The direct reason is that we decided to use structured logging with `logr` for this library.

We decided to go with that because we've been wanting to experiment with moving away from glog for a while.  It has a number of deficiencies (confusing flags, bad defaults, etc), it's not pluggable for people using other logging mechanisms, and it's unstructured.  Structured logging lets us capture useful information in a much-easier-to-parse way.